packages/link/react.d.ts

```diff
@@ -0,0 +1,10 @@
+/// <reference types="react" />
+
+export type Appearance = 'default' | 'subtle'
```

Comment:
What is conventionally exported from a typings file like this?  Everything?

Reply:
That I wasn't quite sure on. I think we could probably leave off Appearance and it would be fine. My experience with typescript type definitions is pretty sparse. Basically just got it to compile :smile:. I'll remove `Appearance` from exports.